packages/manager/apps/pci-quota/tailwind.config.js

```diff
@@ -0,0 +1,13 @@
+import config from '@ovh-ux/manager-tailwind-config';
+
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  ...config,
+  content: [
+    './src/**/*.{js,jsx,ts,tsx}',
+    '../../../manager-react-components/src/**/*.{js,jsx,ts,tsx}',
```

Comment:
since manager-react-component is bundled we don't need to have this config line

but could you add pci-common please ?